ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutorFactory.java

```diff
@@ -131,11 +128,7 @@ static StandaloneExecutor create(
         udfLoader,
         true,
         versionChecker,
-        (ec, sc) -> InjectorChain.of(
-            new DefaultSchemaInjector(
-                new SchemaRegistryTopicSchemaSupplier(sc.getSchemaRegistryClient())),
-            new DefaultTopicInjector(ec)
-        )
+        Injectors.NO_TOPIC_INJECTOR
```

Comment:
Why are we leaving out the topic injector for headless mode? Is this supposed to be `NO_PROPERTY_OVERRIDE_INJECTOR` instead?

Reply:
I have no idea what possessed me...